drivers/nrf_wifi/fw_if/umac_if/inc/offload_raw_tx/fmac_api.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/** @file
+ *
+ * @addtogroup nrf_wifi_api_off_raw_tx FMAC offloaded raw tx API
+ * @{
+ *
+ * @brief Header containing API declarations for FMAC IF Layer of the Wi-Fi driver.
+ */
+
+#ifndef __FMAC_API_H__
+#define __FMAC_API_H__
+
+#include "osal_api.h"
+#include "host_rpu_sys_if.h"
+
+#include "fmac_cmd.h"
+#include "fmac_structs_common.h"
+#include "fmac_api_common.h"
+#include "util.h"
+#include <zephyr/drivers/wifi/nrfwifi/off_raw_tx/off_raw_tx_api.h>
+#include <off_raw_tx.h>
```

Comment:
Is this file needed?

Reply:
Yes it will needed in fmac common api file.